packages/vm/test/util.ts

```diff
@@ -366,7 +366,7 @@ export async function setupPreConditions(state: EVMStateManagerInterface, testDa
   await state.commit()
   // Clear the touched stack, otherwise untouched accounts in the block which are empty (>= SpuriousDragon)
   // will get deleted from the state, resulting in state trie errors
-  ;(<any>state).touchedJournal.clear()
+  //;(<any>state).touchedJournal.clear()
```

Comment:
Intentional?

Reply:
Ah yes, in this method the state is setup via StateManager, not EVM (so there are no touched accounts). Therefore this cleanup action is not necessary. Should remove anyways.